pkg/resource/deploy/deploytest/resourcemonitor.go

```diff
@@ -94,23 +95,28 @@ func (rm *ResourceMonitor) RegisterResource(t tokens.Type, name string, custom b
 	if opts.DeleteBeforeReplace != nil {
 		deleteBeforeReplace = *opts.DeleteBeforeReplace
 	}
+	supportsPartialStables := true
```

Comment:
why is this the default?

Reply:
It's only the default for the test resource monitor--not for the monitor we ship in the CLI.